CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## 1.0.0 (2022-08-29)

```

Comment:
Nice one, we're already introducing breaking changes to the Changelog with the date format.

Reply:
There's only one true date format under the Sun